Project.toml

```diff
@@ -4,15 +4,21 @@ repo = "https://github.com/JuliaSparse/Pardiso.jl.git"
 version = "0.5.5"
 
 [deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MKL_jll = "856f044c-d86e-5d09-b602-aeab76dc8ba7"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
+Compat = "4.10.0"
```

Comment:
Can we just handroll this instead? I really dislike Compat :/.

Reply:
I underastand... We quite well just might export these as it is unlikely that they interfer with other names. Would be the easiest way IMHO.

Reply:
Done this now.